libblkid/src/superblocks/iso9660.c

```diff
@@ -190,8 +190,7 @@ static int probe_iso9660(blkid_probe pr, const struct blkid_idmag *mag)
 
 	blkid_probe_set_block_size(pr, 2048);
 
-	if (!is_str_empty(iso->system_id, sizeof(iso->system_id)))
-		blkid_probe_set_id_label(pr, "SYSTEM_ID",
+	blkid_probe_set_id_label(pr, "SYSTEM_ID",
 				iso->system_id, sizeof(iso->system_id));
 
```

Comment:
Note that blkid_probe_set_id_label() ignores empty ("\0") strings and removes tailing white spaces. It's because on Linux we very often use labels without quotation marks etc.